backend/app/src/user/user.service.ts

```diff
@@ -161,6 +161,117 @@ export class UserService {
     return res.status(200).send();
   }
 
```

Comment:
change PascalCasing to camelCasing:     AffectedUserId: string, =>     affectedUserId: string,